python/dglke/models/general_models.py

```diff
@@ -514,8 +517,13 @@ def forward(self, pos_g, neg_g, gpu_id=-1):
         #    pos_score = (pos_score * subsampling_weight).sum() / subsampling_weight.sum()
         #    neg_score = (neg_score * subsampling_weight).sum() / subsampling_weight.sum()
         #else:
-        pos_score = pos_score.mean()
-        neg_score = neg_score.mean()
+        if self.has_edge_importance:
+            edge_weight = F.copy_to(pos_g.edata['impts'], get_dev(gpu_id))
+            pos_score = (pos_score * edge_weight).sum() / edge_weight.sum()
+            neg_score = (neg_score * edge_weight).sum() / edge_weight.sum()
```

Comment:
`.sum()` works for MXNet as well?

do we need to normalize the score with the edge weight sum?